documentation.yml

```diff
@@ -1,4 +1,6 @@
 toc:
+  - name: Aggregation
```

Comment:
I had moved Aggregation at the bottom because it's not that important (`@turf/collect` is kind of an abstract library) especially as the first library that someone sees.

We should slightly organize the categories at the "Top" to be the easiest to understand and the hardest at the bottom (within the category they can still be alphabetical).